### ISSUE TITLE
Add 3rd test option with the command line flag --load-room

### ISSUE THREAD

**Issue by Zamiell** [Tempus/Basement-Renovator#104]

**Comment by Dgc2002** [comment]
I was working on a PR for this but to be perfectly honest: The current state of the code base isn't a joy to work with. I just called it quits trying to work around some things for now.  
  
I did come up with a working solution though, [here's a gist of it](https://gist.github.com/Dgc2002/ff41f170005009eef77492d5c655877c)

**Comment by Tempus** [comment]
Thanks. The comments on the steam protocol arguments (which I was not aware of) and the notes on crashes caused when loading the room files are very useful. 

I also appreciate the irony of putting down the code base but providing a solution with a hardcoded, windows only path as opposed to modifying the findResourcePath() function which locates the Isaac resources folder with full cross platform support. I know the codebase isn't the greatest - it was started three years ago for Rebirth and got nearly zero interest as each successive expansion was announced. I don't have the interest or time to refactor and abstract further.

**Comment by Dgc2002** [comment]
> I also appreciate the irony of putting down the code base but providing a solution with a hardcoded, windows only path as opposed to modifying the findResourcePath()   
    
And next to that hardcoded path I provided the reasoning for me using it in my personal one-off implementation. `findResourcePath()` doesn't support scenarios where BoI isn't installed in the main steam library. My Steam installation is on my E: drive but my default Steam Library is on my C: drive. Seen here:  

![image](https://user-images.githubusercontent.com/3957791/27257407-d2fe2f0a-539a-11e7-8507-0a950acab22f.png)

(I've triggered myself with the inconsistent naming of the folders now)  
  
> which locates the Isaac resources folder with full cross platform support  

No, it absolutely does not.  
  
`findResourcePath()` does the following:  

* Windows: Grabs a URI from the registry entry `KEY_CURRENT_USER\Software\Valve\Steam\SteamPath` 
  * This URI indicates where **Steam** is installed, not where any games are installed.  
* Mac: Uses a hard coded path    
* Linux: Uses a hard coded path  

The hard coded paths also assume that BoI is installed in a default location. If this isn't the case it won't work.  
  
Could I have modified the `findResourcePath()` function? Of course. But I don't have any environments set up to test Mac or Linux so this would no longer provide 'full cross platform support', any solution that I came up with would only be tested on Windows.  
  
I just decided to try this app in order to turn out a quick couple of rooms for potential use in AI training. I'm likely done using it but saw Zamiell's issue regarding something I'd found a partial solution for, instead of just ignoring it I figured I'd drop off that partial solution.    
  
There's no need to take my comment personally. The comment was regarding the code base's current condition, not you in any way. I don't think I was being harsh either, I just said I didn't enjoy working with it for now. Overall I'm pretty happy with the tool overall and I've recommended it to several people already.
